metabase/metabase
# Add SegmentedToggle component from mantine

Needed for https://github.com/metabase/metabase/issues/35961

### Description

Implements the [Mantine SegmentedControl component](https://v6.mantine.dev/core/segmented-control/) in our metabase/ui component library.

### How to verify

1. `yarn storybook`
2. http://localhost:6006/?path=/story/inputs-segmentedcontrol--default-story , http://localhost:6006/?path=/story/inputs-segmentedcontrol--full-width
3. Confirm it looks and behaves like in provided designs - [here](https://www.figma.com/file/6cHI5jaztfjDYvEXEhy9uE/Setup-flow-for-static%2Fpublic-embedding-%26-embedding-feature-discoverability?type=design&node-id=135-19223&mode=dev) and [here](https://www.figma.com/file/6cHI5jaztfjDYvEXEhy9uE/Setup-flow-for-static%2Fpublic-embedding-%26-embedding-feature-discoverability?type=design&node-id=324-18627&mode=dev) 

### Demo

<img width="200" alt="Screenshot 2023-12-21 at 01 48 45" src="https://github.com/metabase/metabase/assets/7983744/f10004d8-897b-4846-a83c-89e8571b57b4">


### Checklist
